hoomd/md/methods.py

```diff
@@ -57,17 +51,25 @@ class NVT(_Method):
     where :math:`g` is the number of degrees of freedom, and :math:`k_B T_0` is
     the set point (*kT* above).
 
-    *kT* can be a variant type, allowing for temperature ramps in simulation
-    runs.
+    .. rubric:: Rotational degrees of freedom
 
-    A :py:class:`hoomd.compute.thermo` is automatically specified and associated
-    with *group*.
+    `NVT` integrates rotational degrees of freedom. #TODO
```

Comment:
Does `#TODO` work here? rst comments begin with `..`

Reply:
@b-butler  I thought this is general way how dev people can get aware of future tasks. Are we keeping official documentation for todos too?
With legit rst formatting`.. todo:: `, generated html doesn't show the content, which seems good to me.